packages/@react-stately/data/docs/useTreeData.mdx

```diff
@@ -83,10 +88,11 @@ let tree = useTreeData({
   ],
   initialSelectedKeys: ['Sam', 'Kangaroo'],
   getKey: item => item.name,
-  getChildren: item => item.items
+  getChildren: item => item.items || []
```

Comment:
I could add null or undefined to the return values of getChildren instead of doing this, our code is actually safe against that, which is why the previous code didn't blow up

Reply:
I think its reasonable to add `| null` to the getChildren return type in useTreeData since like you mentioned the `buildTree` call that uses `getChildren` defaults initial items to an empty array anyways